### ISSUE TITLE
META.yml lacks `provides`, also missing META.json

### ISSUE THREAD

**Issue by zakame** [damil/Text-Transliterator#2]
I was trying to install your related module Search::Tokenizer, but I could not install it as my CPAN installer could not find Text::Transliterator::Unaccent.

Digging deeper, it appears due to your [META.yml][0] of this dist on the CPAN is lacking a [`provides` section][1] describing the modules in the dist, which although optional, is needed for the CPAN indexers to find the modules you wish to be used from your dist.  Curiously, the [META.yml][2] in this repo does have `provides`, but out of date (and seems to be listing a wrong version of Text::Transliterator::Unaccent as well.)

[0]: https://metacpan.org/source/DAMI/Text-Transliterator-1.02/META.yml
[1]: https://metacpan.org/pod/CPAN::Meta::Spec#provides
[2]: https://github.com/damil/Text-Transliterator/blob/master/META.yml

This also affects the appearance of Text::Transliterator::Unaccent as a just a "Documentation" module in the [MetaCPAN][3].

[3]: https://metacpan.org/release/DAMI/Text-Transliterator-1.02

Lastly, `META.json` is missing in the repository, and modern versions of Module::Build now warn about it; a quick `./Build distmeta` is probably needed.

This is likely related to #1 as well.

**Comment by zakame** [comment]
Further info: installing a on clean Docker container, `cpanm Search::Tokenizer` will fail to find the latest version of Text::Transliterator, falling back to installing BackPAN version (1.01) instead:

```
[zakame:~] 31m5s % docker run --rm -it perl:5.24 /bin/sh   
# cpanm Search::Tokenizer                               
--> Working on Search::Tokenizer
Fetching http://www.cpan.org/authors/id/D/DA/DAMI/Search-Tokenizer-1.01.tar.gz ... OK
==> Found dependencies: Module::Build
--> Working on Module::Build
Fetching http://www.cpan.org/authors/id/L/LE/LEONT/Module-Build-0.4224.tar.gz ... OK
Configuring Module-Build-0.4224 ... OK
Building and testing Module-Build-0.4224 ... OK
Successfully installed Module-Build-0.4224
Configuring Search-Tokenizer-1.01 ... OK
==> Found dependencies: Unicode::CaseFold, Text::Transliterator::Unaccent
--> Working on Unicode::CaseFold
Fetching http://www.cpan.org/authors/id/A/AR/ARODLAND/Unicode-CaseFold-1.00.tar.gz ... OK
Configuring Unicode-CaseFold-1.00 ... OK
Building and testing Unicode-CaseFold-1.00 ... OK
Successfully installed Unicode-CaseFold-1.00
! Finding Text::Transliterator::Unaccent on cpanmetadb failed.
--> Working on Text::Transliterator::Unaccent
Fetching http://backpan.perl.org/authors/id/D/DA/DAMI/Text-Transliterator-1.01.tar.gz ... OK
Configuring Text-Transliterator-1.01 ... OK
Building and testing Text-Transliterator-1.01 ... OK
Successfully installed Text-Transliterator-1.01
Use of literal non-graphic characters in variable names is deprecated at /usr/local/lib/perl5/site_perl/5.24.1/Text/Transliterator.pm line 26, <$handle> line 6.
Building and testing Search-Tokenizer-1.01 ... OK
Successfully installed Search-Tokenizer-1.01
4 distributions installed
```

**Comment by damil** [comment]
Fixed in v1.03
